kahuna/public/js/util/crop.js

```diff
@@ -24,7 +19,7 @@ cropUtil.constant('video', video);
 cropUtil.constant('square', square);
 cropUtil.constant('freeform', freeform);
 cropUtil.constant('cropOptions', cropOptions);
-cropUtil.constant('defaultCrop', landscape);
+cropUtil.constant('defaultCrop', cropOptions[0]);
```

Comment:
I think this should stay as is, as the fact that the default is 'landscape' is more explicit/readable
```suggestion
cropUtil.constant('defaultCrop', landscape);
```

Reply:
but if you don't want landscape as default, then ignore this suggestiong and stick with `cropOptions[0]`

Reply:
The issue here is that BBC doesn't want the landscape option, and the default will be a BBC specific one. That's why I set it to the first option in cropOptions (which in BBC will be a different option, not landscape). 

Perhaps we could add a flag inside the options that says "default: true" and then check on that to make it more explicit. What do you think?

Reply:
thats a nice idea, but don't worry, I think first item in the list is sufficient, probs not worth the extra complexity (in ensuring that its set on one [and only one] entry)